features/step_definitions/work_steps.rb

```diff
@@ -454,3 +454,9 @@
 Then /^I should not find a list for associations$/ do
   page.should_not have_xpath("//ul[@class=\"associations\"]")
 end
+
+Then /^the work "([^\"]*)" should be deleted$/ do |work|
+  assert Work.where(title: work).count == 0
```

Comment:
@shalott BTW, for these kinds of queries, you can do

    Work.where(title: work).exists?

instead, which is a little more efficient because it doesn't do a count query.